docs/reference/announcements.md

```diff
@@ -132,6 +132,27 @@ Starting with Camunda 8.6, the Camunda Optimize artifact has been split into two
 
 Make sure to update your Docker configurations accordingly to ensure compatibility.
 
+### New base path for Operate and Tasklist Webapps
+
+We are introducing a new base path for both the Operate and Tasklist **web applications**. This change applies to both self-managed and SaaS environments.
```

Comment:
Should this content also be added to the Operate and Tasklist documentation?

Reply:
I did not find a good place where to describe these changes.
FYI, the impact should be minimal to the end users, the old urls should work and directed to the new urls with base paths. Also, this can be subject to other changes in the future as we working on it in scope of C8 streamlining.

Reply:
Can always find a spot to incorporate this if we get any feedback from users, but I'm fine with this 👍